### ISSUE TITLE
[Bug] Nether teleport location wrong

### ISSUE THREAD

**Issue by voodootje0** [rlf/uSkyBlock#534]
![image](https://cloud.githubusercontent.com/assets/6852308/10868740/e3356c76-8097-11e5-9faa-dc367480310e.png)

From my island portal I teleported to the that portal.

**Comment by voodootje0** [comment]
Now I reset my island and this is my nether island.
![image](https://cloud.githubusercontent.com/assets/6852308/10868743/2145f3aa-8098-11e5-897b-2b0a7b59560d.png)

I also teleported there.

**Comment by voodootje0** [comment]
The third reset my island is normal.

![image](https://cloud.githubusercontent.com/assets/6852308/10868744/424181dc-8098-11e5-819b-3b7c856d4fb6.png)

**Comment by voodootje0** [comment]
The fourth time this is my island.
![image](https://cloud.githubusercontent.com/assets/6852308/10868754/6501eacc-8098-11e5-8be2-61f3ad615d0e.png)

**Comment by rlf** [comment]
Questions:
a) When do you teleport to your island?
b) the 3 screendumps, are you staying in the nether while testing (flying?)

Because:
a) The nether island is pasted using WE (AWE), same time as the "overworld island", so if you teleport to it too fast, the portal has not yet been formed.
b) The nether island is wiped on `/is restart` using world-unload, world-generate, which means you shouldn't be there when it happens - since your client and the server world will be out of sync (the reason we teleport players to spawn).

**Comment by EvilOlaf** [comment]
Confirmed....well partially.

What I did after installation and a second restart to let uSkyBlock create the netherworld was using creative mode to help myself to some obsidian and a lighter, build a portal and get through. I had exactly the same scenario as visible above.

I tried again with `/is restart` and it worked.

Tried to delete the skyworld_nether while server is stopped and reproduce but it kept working as expected.
